include/RAJA/index/IndexSetUtils.hpp

```diff
@@ -113,10 +108,10 @@ RAJA_INLINE void getIndicesConditional(CONTAINER_T& con,
 {
   CONTAINER_T tcon;
   forall<seq_exec>(seg,
-    [&](typename CONTAINER_T::value_type idx) {
-      if (conditional(idx)) tcon.push_back(idx);
-    }
-  );
+                   [&](typename CONTAINER_T::value_type idx)
+                   {
+                     if (conditional(idx)) tcon.push_back(idx);
+                   });
```

Comment:
Is this the expected result ? (The body indentation depends on the function name length, which is odd).

Reply:
I think that is due to turning on vertical alignment of function args, which can be argued either way whether it's an improvement.